packages/react-composites/src/composites/CallComposite/Strings.tsx

```diff
@@ -268,4 +268,14 @@ export interface CallCompositeStrings {
    * Label for the hold pane
    */
   holdScreenLabel: string;
+  /* @conditional-compile-remove(PSTN-calls) */
+  /**
+   * Placeholder text for dtmf dialpad
+   */
+  dtmfDialpadPlaceHolderText: string;
+  /* @conditional-compile-remove(PSTN-calls) */
+  /**
+   * Label for the button to open dtmf dialpad
+   */
+  openDtmfDialpad: string;
```

Comment:
Nomenclature in this file is to suffix with `Label` for button labels

Reply:
Sorry missed this comment! Will change this in the next PR